etc/kayobe/kolla.yml

```diff
@@ -331,6 +335,88 @@ kolla_build_blocks:
     {% set magnum_capi_packages = ['git+https://github.com/stackhpc/magnum-capi-helm.git@v0.13.0'] %}
     RUN {{ macros.install_pip(magnum_capi_packages | customizable("pip_packages")) }}
     {% endraw %}
+  prometheus_alertmanager_repository_version: | # 2023.1 kolla has 0.24.0
+    {% raw %}
+    ARG prometheus_alertmanager_version=0.26.0
+    ARG prometheus_alertmanager_archive=alertmanager-${prometheus_alertmanager_version}.linux-{{debian_arch}}.tar.gz
+    ARG prometheus_alertmanager_sha256sums_url=https://github.com/prometheus/alertmanager/releases/download/v${prometheus_alertmanager_version}/sha256sums.txt
+    ARG prometheus_alertmanager_download_url=https://github.com/prometheus/alertmanager/releases/download/v${prometheus_alertmanager_version}/${prometheus_alertmanager_archive}
```

Comment:
nit: You could have used kolla_build_args to override only prometheus_alertmanager_version.

Reply:
I tried that but it was not overriding the arguments unfortunately